outline.md

```diff
@@ -0,0 +1,45 @@
+documentation outline:
```

Comment:
This looks more suitable for the wiki, or maybe a master issue for improving docs.

Reply:
yes, maybe you can put this in a wiki

Reply:
Thanks, yes I forgot it here. That's right, I'll open a page on the wiki for that